SampleApplications/SDK/Opc.Ua.Server/NodeManager/MasterNodeManager.cs

```diff
@@ -1682,8 +1720,9 @@ public void RemoveReferences(List<LocalReference> referencesToRemove)
 
             // clear the diagnostics array if no diagnostics requested or no errors occurred.
             UpdateDiagnostics(context, diagnosticsExist, ref diagnosticInfos);
-        }      
-             
+        }
+        #region Validate Role Permissions
+        #endregion
```

Comment:
empty artifact?

Reply:
Fixed. Code was moved to #region Validate Permissions Methods and that one remained empty